src/app/views/route-search-results/route-search-results.component.ts

```diff
@@ -80,7 +83,7 @@ export class RouteSearchResultsComponent implements OnInit {
   }
 
   getThresholdCo2(): number {
-    let max: number = 0;
```

Comment:
@bucherfa thats not commit message related :P

Reply:
yea, just what my editor wanted from me ;)